yt_dlp/extractor/rumble.py

```diff
@@ -179,29 +195,32 @@ def _real_extract(self, url):
                 meta = video_info.get('meta') or {}
                 if not video_info.get('url'):
                     continue
+                # With default query parms returns m3u8 varients which are duplicates, without returns tar files
+                if ext == 'tar':
+                    continue
                 if ext == 'hls':
                     if meta.get('live') is True and video.get('live') == 1:
                         live_status = 'post_live'
                     formats.extend(self._extract_m3u8_formats(
                         video_info['url'], video_id,
                         ext='mp4', m3u8_id='hls', fatal=False, live=live_status == 'is_live'))
                     continue
-                timeline = ext == 'timeline'
-                if timeline:
-                    ext = determine_ext(video_info['url'])
+                is_timeline = ext == 'timeline'
+                is_audio = ext == 'audio'
                 formats.append({
-                    'ext': ext,
-                    'acodec': 'none' if timeline else None,
+                    'ext': determine_ext(video_info['url']),
```

Comment:
Unnecessary; core code does this. The old code would use the `ext` var here if it wasn't "timeline" -- should we do the same if it isn't "timeline" or "audio"?
```suggestion
                    'ext': ext if not (is_timeline or is_audio) else None,
```

Reply:
```suggestion
```